SI/Sem.07/Pract.07/References/Pract07-04.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+using namespace std;
+
+void SwapValuesIfHigher(int& n, int& m)
+{
+	if (n > m)
+	{
+		int copyN = n;
+		n = m;
+		m = copyN;
+	}
+}
+
+int FindGCD(int lower, int higher)
+{
+	int gcd = 0;
+	SwapValuesIfHigher(lower, higher);
+	for (int i = 1; i <= higher; i++)
+	{
+		if (lower % i == 0 && higher % i == 0)
+		{
+			gcd = i;
+		}
+	}
+	return gcd;
+}
```

Comment:
There are much better algorithms for gcd, think what you have leant in the workshops.